network-area-diagram/src/main/java/com/powsybl/nad/svg/AbstractStyleProvider.java

```diff
@@ -84,11 +84,19 @@ public List<String> getSideEdgeStyleClasses(BranchEdge edge, BranchEdge.Side sid
     @Override
     public List<String> getEdgeInfoStyleClasses(EdgeInfo info) {
         List<String> styles = new LinkedList<>();
-        switch (info.getInfoType()) {
-            case EdgeInfo.ACTIVE_POWER -> styles.add(CLASSES_PREFIX + "active");
-            case EdgeInfo.REACTIVE_POWER -> styles.add(CLASSES_PREFIX + "reactive");
-            case EdgeInfo.CURRENT -> styles.add(CLASSES_PREFIX + "current");
-            default -> LOGGER.warn("The \"{}\" type of information is not handled", info.getInfoType());
+        String externalInfoType = info.getInfoType2();
```

Comment:
I don't see how we could deal with style here if there are two info types. The arrow takes the reference value for the style, but how do we deal different colors for different text type (that could lead to different colors for text on both sides)?